Framework/DataHandling/src/LoadILLDiffraction.cpp

```diff
@@ -258,6 +258,7 @@ void LoadILLDiffraction::loadMetaData() {
     m_loadHelper.addNexusFieldsToWsRun(nxHandle, m_outWorkspace->mutableRun());
     NXclose(&nxHandle);
   }
+  mutableRun.addProperty("run_list", std::string(mutableRun.getProperty("run_number")->value()));
```

Comment:
`getPropertyValue()` should return you a string directly.

Reply:
`getPropertyValue()` is not a member function of  `Run` nor `LogManager` classes, it is possible to extract property values as strings from algorithms this way, but not from the mutable run object here. I agree the current solution is a bit of a mouthful, and perhaps a better alternative would be to use `getPropertyValueAsType<std::string>`.